### PR TITLE
add zh_CN translation

### PR DESCRIPTION
**Pull request by jonah2022**
HI, I wrote some po file for a new pull request.
this po is translate for lib/common/cmdline.c

### CODE REVIEW

**Comment by wenningerk** [comment]
test this please

File: lib/common/cmdline.c [hunk 1]
```diff
@@ -89,15 +89,15 @@ pcmk__build_arg_context(pcmk__common_args_t *common_args, const char *fmts,
         GOptionEntry output_entries[3] = {
             { "output-as", 0, 0, G_OPTION_ARG_STRING, &(common_args->output_ty),
               NULL,
-              "FORMAT" },
+              N_("FORMAT") },
             { "output-to", 0, 0, G_OPTION_ARG_STRING, &(common_args->output_dest),
-              "Specify file name for output (or \"-\" for stdout)", "DEST" },
+             N_( "Specify file name for output (or \"-\" for stdout)"),N_("DEST") },
```
Comment (gao-yan) [inline lib/common/cmdline.c:94]: Rather a nitpicking than a real problem, but could we keep the whitespace after the `,`  in between for the sake of readability? This really is a big struct with a long list of variables :-)

**Review by gao-yan — COMMENTED** [review]
LGTM. Only a nitpicking.

File: lib/common/cmdline.c (hunk 1 above)
Comment (jonah2022) [reply lib/common/cmdline.c:94]: ok,I will fix it:-)

**Comment by wenningerk** [comment]
could you pls squash the commits (typo in commit-name anyway ;-) )

**Comment by jonah2022** [comment]
> 

squashed :-)

**Review by gao-yan — APPROVED** [review]
Thanks.

**Comment by kgaillot** [comment]
Thanks!

**Pull request merged**